src/Microsoft.FileFormats/PDB/PDBFile.cs

```diff
@@ -8,75 +8,60 @@ namespace Microsoft.FileFormats.PDB
 {
     public class PDBFile : IDisposable
     {
-        private readonly Reader _reader;
-        private readonly Lazy<PDBFileHeader> _header;
-        private readonly Lazy<Reader[]> _streams;
+        private readonly IMSFFile _msfFile;
         private readonly Lazy<PDBNameStream> _nameStream;
         private readonly Lazy<DbiStream> _dbiStream;
 
-        public PDBFile(IAddressSpace dataSource)
+        private PDBFile(IMSFFile msfFile)
```

Comment:
Yeah, it's an incomplete rebase from a different repo.  Will fix, obviously.  The intent is to use `PDBFile.Open` instead of directly calling the constructor.

Reply:
You still can't change/remove an existing public function in this assembly.  There is code in this repo and outside of it that references this constructor.

Reply:
Thanks, Mike.  I've restored the old constructor, the `Streams` property, etc. It should provide exactly the same API as before, in addition to newer methods that are more efficient (directly calling `GetStream`).